config/user-libblkid.m4

```diff
@@ -16,20 +16,13 @@ dnl # Otherwise we disable blkid support and resort to manual probing.
 dnl #
 AC_DEFUN([ZFS_AC_CONFIG_USER_LIBBLKID], [
 	AC_ARG_WITH([blkid],
-		[AS_HELP_STRING([--with-blkid],
-		[support blkid caching @<:@default=check@:>@])],
+		[AS_HELP_STRING([--without-blkid],
+		[disable suport for  blkid])],
 		[],
-		[with_blkid=check])
+		[with_blkid=yes])
 
 	LIBBLKID=
-	AS_IF([test "x$with_blkid" = xyes],
-	[
-		AC_SUBST([LIBBLKID], ["-lblkid"])
-		AC_DEFINE([HAVE_LIBBLKID], 1,
-			[Define if you have libblkid])
-	])
-
-	AS_IF([test "x$with_blkid" = xcheck],
+	AS_IF([test "x$with_blkid" != xno],
```

Comment:
This was intentionally set to xcheck because the test program executes code in libblkid that touches files prohibited by Gentoo's sandbox during the configure script. Using `blkid` to check, I think the issue was this:

```
open("/run/blkid/blkid.tab-CBSeOf", O_RDWR|O_CREAT|O_EXCL|O_CLOEXEC, 0600) = 3
```

I am not sure if Gentoo QA will allow me to add an exception. It might be necessary to try to detect the sandbox to disable this solely on Gentoo.